ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ValidateSchema.java

```diff
@@ -34,11 +66,18 @@ private static void loadSchema(String name) throws SchemaNotFoundException {
       JSONObject rawSchema = Json.readJSONObject(schema);
       schemas.put(name, SchemaLoader.load(rawSchema));
     } catch (IOException e) {
-      // TODO load all existing schemas on startup
-      throw new SchemaNotFoundException("No schema with name:" + name);
+      throw new SchemaNotFoundException(name);
     }
   }
 
+  private static Schema getSchema(String name) throws SchemaNotFoundException {
+    final Schema schema = schemas.get(name);
+    if (schema == null) {
+      throw new SchemaNotFoundException(name);
+    }
+    return schema;
+  }
```

Comment:
Added `getSchema(String)` so that we can still throw SchemaNotFoundException if the given schema is not present in the map.